homeassistant/components/heos/__init__.py

```diff
@@ -87,9 +87,7 @@
             favorites = await controller.get_favorites()
         else:
             _LOGGER.warning(
-                "%s is not logged in to a HEOS account and will be unable "
-                "to retrieve HEOS favorites: Use the 'heos.sign_in' service "
-                "to sign-in to a HEOS account",
+                "%s is not logged in to a HEOS account and will be unable to retrieve HEOS favorites: Use the 'heos.sign_in' service to sign-in to a HEOS account",
```

Comment:
This long like isn't an improvement in readability.

Reply:
This is a limitation currently in black. There's an open feature request to split long strings.

We used to handle this with max line length check in flake8/pylint.

Until there's a better solution, I don't think we should deliberately push the line length up, even though there's no tooling in place to stop us.

Reply:
The problem is that we didn't anticipate that some code wouldn't be handled by black. We shouldn't try to exploit loopholes even though they are there.

Black aims at keeping lines limited at 88 characters. I think we should do the same until we have tooling in place.